packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -66,6 +64,15 @@ const argv = yargs
   .help()
   .argv;
 
+const WATCH_EXPRESSION = [
```

Comment:
This should probably become a function, then: takes in an `options` argument with `{extensions: Array<string>}` as the only option for now. Outputs this watchman expression.

```
function buildWatchExpression(options: {extensions: Array<string>} {
  return [
    ...
    ['anyof', extensions.map(ext => ['suffix', ext])],
    ...
  ]
}
```